develop/dev-guide-choose-driver-or-orm.md

```diff
@@ -95,15 +101,15 @@ implementation group: 'org.bouncycastle', name: 'bcpkix-jdk15on', version: '1.67
 
 ### Java ORM framework
 
-#### Hibernate
-
-Support level: `Full`
-
 > **Note:**
 >
-> Currently, Hibernate does [not support nested transactions](https://stackoverflow.com/questions/37927208/nested-transaction-in-spring-app-with-jpa-postgres). Since v6.2.0, TiDB supports [savepoint](/sql-statements/sql-statement-savepoint.md).
->
-> If you are using a framework such as `Spring Data JPA`, do not use the `Propagation.NESTED` transaction propagation option in `@Transactional`, that is, do not set `@Transactional( propagation = Propagation.NESTED)`.
+> - Currently, Hibernate does [not support nested transactions](https://stackoverflow.com/questions/37927208/nested-transaction-in-spring-app-with-jpa-postgres).
+> - Since v6.2.0, TiDB supports [savepoint](/sql-statements/sql-statement-savepoint.md). Therefore, make sure that your TiDB is v6.2.0 or later versions. Otherwise, do not use the `Propagation.NESTED` transaction propagation option in `@Transactional`, that is, do not set `@Transactional(propagation = Propagation.NESTED)`.
```

Comment:
```suggestion
> - Since v6.2.0, TiDB supports [savepoint](/sql-statements/sql-statement-savepoint.md). To use the `Propagation.NESTED` transaction propagation option in `@Transactional`, that is, to set `@Transactional(propagation = Propagation.NESTED)`, make sure that your TiDB is v6.2.0 or later.
```